internal/servers/controller/handlers/hosts/host_service.go

```diff
@@ -134,6 +134,8 @@ func (s Service) ListHosts(ctx context.Context, req *pbs.ListHostsRequest) (*pbs
 		switch h := item.(type) {
 		case *plugin.Host:
 			outputOpts = append(outputOpts, handlers.WithHostSetIds(h.SetIds))
+		case *static.Host:
```

Comment:
Since all host types now support setIds it seems we should extend the Host interface defined [here](https://github.com/hashicorp/boundary/blob/main/internal/host/host.go#L24) to include a `GetSetIds() []string` method. That way we can skip this type switch and just do:
```
outputOpts = append(outputOpts, handlers.WithHostSetIds(item.GetSetIds()))
```